hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/TransactionChecker.java

```diff
@@ -221,7 +209,11 @@ public TransactionInfo check(@NonNull final Transaction tx, @Nullable Bytes seri
         final SignatureMap signatureMap;
         if (tx.hasBody()) {
             txBody = tx.bodyOrThrow();
-            bodyBytes = TransactionBody.PROTOBUF.toBytes(txBody);
+            try {
```

Comment:
There is a place in `TransactionParts` that doesn't use `ProtobufUtils` to parse. But that can be fixed in a different PR.

Reply:
@Neeharika-Sompalli: Is `TransactionParts` used in places where we need the original bytes?

Reply:
`TransactionParts` is only used in test-clients, so we don't need to worry much about that.